README.md

```diff
@@ -57,6 +62,10 @@ then:
 product([f1, ... , fn]): [A1, ... , An] ~> [P1, ... , Pn]
 ```
 
+in short:
+
+We use the operator *product* whenever we want (and we always do if we can) to run multiple fetches in **parallel**.
```

Comment:
Right, like this people may think "I use *product* when I want to improve perfs" instead of "I use *product* when I have a set of independent fetches (perf is a side-effect)

I'll improve it